repository_service_tuf/cli/admin/sign.py

```diff
@@ -125,7 +122,7 @@ def sign(
 
     # Make sure user understands that result will be send to the API and if the
     # the user wants something else should use '--dry-run'.
-    if settings.get("SERVER") is None and not dry_run:
+    if not settings.get("SERVER") and not dry_run:
         raise click.ClickException(
             "Either '--api-sever' admin option/'SERVER' in RSTUF config or "
```

Comment:
```suggestion
            "Either '--api-server' admin option/'SERVER' in RSTUF config or "
```
I see this typo several times :)
Not related to your PR, but I will open a issue

Reply:
Will fix it now.

Reply:
Fixe